dependency_management/set_lower_bounds

```diff
@@ -42,15 +42,18 @@ def _do_updates(explicit_dependencies, cargo_tree):
     update_failed = False
     for (crate, version) in explicit_dependencies.items():
         current_version = cargo_tree[crate]
+        command = [
```

Comment:
Because the command got pulled out of the ```subprocess.run``` call it's time to hoist the non-changing parts right out of the loop entirely.

Above the loop, set something called, e.g., ```command_prefix``` to all the non-changing parts of the command.

Then, within the loop:

```command = command_prefix + ["--package=%s:%s" % (crate, current_version), "--precise=%s" % version]```

because ```--package``` and ```--precise``` argument are the only arguments that actually change within the loop.

Reply:
Resolved.